.changelog/1179.bugfix.md

```diff
@@ -0,0 +1 @@
+Update home page "metaverse" graph for consensus
```

Comment:
Adding new Consensus functionality is trivial task rather than bugifx

Reply:
OK renamed.